lib/context/provider-context.tsx

```
@@ -0,0 +1,79 @@
+      setProviderData(null);
+      return;
+    }
+
+    setIsLoading(true);
+    setError(null);
+
+    try {
+      const apiClient = await getApiClient();
+      const response = await apiClient.get('/providers/me');
+      setProviderData(response.data);
+    } catch (err: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)